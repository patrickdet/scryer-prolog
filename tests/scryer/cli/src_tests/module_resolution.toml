args = [
    "-f",
    "--no-add-history",
    "src/tests/module_resolution.pl",
    "-f",
    "-g", "use_module(library(module_resolution))",
    "-g", "get_module(some_predicate, M), write(M), write('\\n')",
    "-g", "module_resolution:get_module(some_predicate, M), write(M), write('\\n')",
    "-g", "halt"
]
