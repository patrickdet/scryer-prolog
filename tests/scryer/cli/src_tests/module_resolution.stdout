module_resolution
module_resolution
module_resolution
module_resolution
user
user
